cmd/sidecars/launcher/launcher.go

```diff
@@ -0,0 +1,58 @@
+package launcher
```

Comment:
I'm wondering about the name. If we want to share more code in the future between sidecar-shim and other sidecars, launcher could be odd but it is perfect for now... Naming is hard.